assets/scss/00-base/_fonts-langs-import.scss

```diff
@@ -0,0 +1,152 @@
+@use "variables" as *;
+// Variable sans fonts
+$adlam:						U+1E900-1E95F, U+2E41; // Uses reversed comma
+$arabic:					U+0600-06FF, U+0750-077F, U+08A0-08FF, U+FB50-FDFF, U+FE70-FEFF, U+1EE00-1EEFF; // Arabic, supplement, extended A, presentation forms A and B, alphabetic mathematical symbols
+$armenian:				U+0530-058F;
+$balinese:				U+1B00-1B7F;
+$bamum:						U+A6A0-A6FF, U+16800-16A3F; // Bamum and supplement
+$bengali:					U+0980-09FF;
+$brahmi:					U+11000-1107F;
+$buginese:				U+1A00-1A1F;
+$buhid:						U+1740-175F, U+1735;
+$c-aboriginal:		U+1400-167F, U+18B0-18FF; // UCAS and extended
+$cham:						U+AA00-AA5F;
+$cherokee:				U+????; // U+13A0-13FF;
+$chinese:					U+4E00-9FFF, U+31C0-31EF, U+3300-33FF, U+3000-303F, U+2E80-2EFF, U+FE30-FE4F, U+2F00-2FDF, U+FF0C;
+$cyrillic:				U+0400-04FF, U+0500-052F, U+1C80-1C8F;
+$devanagari:			U+0900-097F, U+A8E0-A8FF; // Devanagari and extended
+$elemaic:					U+10FE0-10FFF;
+$ethiopic:				U+1200-137F, U+1380-139F, U+2D80-2DDF; // Ethiopic, extended, and supplement
+$grantha:					U+11300-1137F;
+$georgian:				U+10A0-10FF, U+1C90-1CBF, U+2D00-2D2F; // Georgian, extended, and supplement
+$greek:						U+0370-03FF, U+1F00-1FFF; // Greek and coptic, greek extended
+$gujarati:				U+0A80-0AFF;
+$gurmukhi:				U+0A00-0A7F;
+$hangul:					U+1100-11FF, U+3130-318F, U+AC00-D7AF, U+A960-A97F, U+D7B0-D7FF; // With compatibility jamo
+$hanifi-rohingya:	U+10D00-10D3F;
+$hanunoo:					U+1720-173F;
+$hebrew:					U+0590-05FF, U+FB1D-FB4F;
+$japanese:				U+A5, U+4E00-9FFF, U+30??, U+FF00-FF9F; // Kanji, Hiragana, Katakana, Yen symbol.
+$kannada:					U+0C80-0CFF;
+$kayah:						U+A900-A92F;
+$khmer:						U+1780-17FF, U+19E0-19FF; // Khmer and symbols
+$lao:							U+0E80-0EFF;
+$latin:						U+????;
+$limbu:						U+1900-194F;
+$lisu:						U+A4D0-A4FF;
+$malayalam:				U+0D00-0D7F;
+$myanmar:					U+1000-109F, U+AA60-AA7F, U+A9E0-A9FF;
+$mongolian:				U+1800-18AF, U+11660-1167F;
+$nko:							U+07C0-07FF;
+$nushu:						U+1B170-1B2FF;
+$oriya:						U+0B00-0B7F;
+$olchiki:					U+1C50-1C7F;
+$rashi-hebrew:		U+0590-05FF;
+$sinhala:					U+0D80-0DFF;
+$sora-sompeng:		U+110D0-110FF;
+$tai-le:					U+1950-197F;
+$tai-tham:				U+1A20-1AAF;
+$tai-viet:				U+AA80-AADF;
+$takri:						U+11680-116CF;
+$tamil:						U+0B80-0BFF, U+11FC0-11FFF; // Tamil and supplement
+$telugu:					U+0C00-0C7F;
+$tagalog:					U+1700-171F, U+1735;
+$thaana:					U+0780-07BF;
+$thai:						U+0E00-0E7F;
+$tibetan:					U+0F00-0FFF;
+$warang-citi:			U+118A0-118FF;
+$zanabazar:				U+11A00-11A4F;
+$lgc: 						U+0000-00FF, U+0100-017F, U+0180-024F, U+0370-03FF, U+0259-03C0, U+1E00-1EFF, U+2000-FB02, U+1F00-1FFF, U+0400-04FF, U+0500-052F, U+1C80-1C8F;
+
+
+$oneSpace: 	' ';
+$oneDash:		'-';
+$global-font-display:	'swap';
+
+@mixin font-face ($family: "Noto", $type: "Sans", $script: "Cyrillic", $suffix: "UI", $weight: "VF", $subset: "-subset", $fontWeight: "100 900", $fontStyle: "normal", $fontDisplay: "swap", $unicodeRange: "cyrillic", $space: $oneSpace, $dash: $oneDash) {
```

Comment:
Move this mixin to 00-base/mixins/_font-face.scss and then add a `@use '00-base/mixins/font-face'`;